invertedai/utils.py

```diff
@@ -443,17 +443,26 @@ def _interpret_response_line(self, result):
         return data
 
 @validate_call
-def get_default_agent_attributes(agent_count_dict: Dict[str,int]) -> List[AgentAttributes]:
+def get_default_agent_properties(
+    agent_count_dict: Dict[str,int],
+    attribute_type: Optional[str] = "properties"
```

Comment:
If this is meant to be a flag to by default use or not use `agent_properties`, change it to boolean `use_agent_properties` with a default value. Also update the doc string below accordingly.

Reply:
Done